RULES.md

```diff
@@ -92,7 +92,7 @@ The [AWS Solutions Library](https://aws.amazon.com/solutions/) offers a collecti
 | AwsSolutions-IAM4  | The IAM user, role, or group uses AWS managed policies.                                                                                                                                   | An AWS managed policy is a standalone policy that is created and administered by AWS. Currently, many AWS managed policies do not restrict resource scope. Replace AWS managed policies with system specific (customer) managed policies.                                                                                                                                                                                                                                                                                                                              |
-| AwsSolutions-IAM5  | The IAM entity contains wildcard permissions and does not have a cdk_nag rule suppression with evidence for those permission.                                                             | Metadata explaining the evidence (e.g. via supporting links) for wildcard permissions allows for transparency to operators.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| AwsSolutions-IAM5  | The IAM entity contains wildcard permissions and does not have a cdk_nag rule suppression with evidence for those permission.                                                             | Metadata explaining the evidence (e.g. via supporting links) for wildcard permissions allows for transparency to operators. This is a granular rule that returns indivisual findings that can be suppressed with `appliesTo`. The findings are in the format `Action::<action>` for policy actions and `Resource::<resource>` for resources. Example: `appliesTo: ['Action::s3:*']`.                                                                                                                                                                                   |
```

Comment:
This should also be reflected [here](https://github.com/tkglaser/cdk-nag/blob/d0901a521e2c042075e3d0666589de9cc2659b90/src/packs/aws-solutions.ts#L1209)